bin/oref0-setup.sh

```diff
@@ -70,6 +75,10 @@ case $i in
     ;;
 esac
 done
+if [[ -z $TTY && radio_locale != "US" ]]; then
```

Comment:
This will only work right if you define radio_locale to be "US" by default above.